### PR TITLE
Cache tokenizer

### PR DESCRIPTION
**Pull request by JosephMarinier**
## Description:

Cache Hugging Face files when using Docker Compose. Hugging Face cache includes tokenizer in `transformers`, as well as `datasets`, `metrics` and `modules`. The default value is `~/.cache/hugginface`. Instead, we set it to `/azimuth_shr/huggingface_cache`, which is mounted.

## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **FRONTEND TYPES.** Regenerate the front-ent types if you played with types and routes.
  Run `cd webapp && yarn types` while the back-end is running.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

**Review by gabegma — COMMENTED** [review]
Can you explain why is this a different path than the home - aren't they the same? Can we keep the same naming convention for the env var? `SENTENCE_TRANSFORMERS_X`?

File: docker-compose.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ services:
     environment:
       - "CFG_PATH=${CFG_PATH}"
       - "SENTENCE_TRANSFORMERS_HOME=/azimuth_shr/sentence_transformers"
+      - "TRANSFORMERS_CACHE=/azimuth_shr/sentence_transformers_cache"
```
Comment (JosephMarinier) [inline docker-compose.yml:8]: @gabegma In reverse order:
> Can we keep the same naming convention for the env var? `SENTENCE_TRANSFORMERS_X`?

I have no control over the name `TRANSFORMERS_CACHE`. It is defined [here](https://github.com/huggingface/transformers/blob/47e1676255e5dd86b9541f734cd4f4bdcbb50f4a/src/transformers/utils/hub.py#L94) on the Hugging Face side, and mentioned [here](https://huggingface.co/docs/transformers/installation?highlight=transformers_cache#cache-setup) in the documentation.

As for `SENTENCE_TRANSFORMERS_HOME`, I guess I don't have control, but to be honest, I don't know where it comes from. Do you?

> Can you explain why is this a different path than the home - aren't they the same?

I guess the two could have the same path value. I see some examples like [this](https://github.com/huggingface/hub-docs/blob/7c236b177759e523f6227367b33ac3f936b19638/api-inference-community/docker_images/sentence_transformers/Dockerfile#L20-L21) in Hugging Face documentation where the two have the same path value.

Comment (gabegma) [reply docker-compose.yml:8]: Ah if this is standard, then that sounds good! Let's keep it as is.

**Review by gabegma — APPROVED** [review]
LGTM!

File: docker-compose.yml (hunk 1 above)
Comment (JosephMarinier) [reply docker-compose.yml]: Would you like to re-review with my updated solution? I updated the PR description for more details.

**Review by gabegma — APPROVED** [review]
LGTM!

**Pull request merged**